tests/core/language_model/test_hf_lm.py

```diff
@@ -211,17 +211,22 @@ def test_if_random_seed_fixes_the_lm_outputs(lm_init_func: Callable[..., Hugging
     assert len(completions) > 1
 
 
-def test_batch_generate_chat_response(lm: LanguageModel) -> None:
-    responses = lm.batch_generate_chat_response([[{"role": "user", "content": "こんにちは。"}]], max_length=40)
+@pytest.fixture(scope="module")
+def chat_lm(model_name: str = "sbintuitions/tiny-lm-chat") -> HuggingFaceLM:
+    return HuggingFaceLM(model=model_name, model_kwargs={"torch_dtype": "float32"})
+
+
+def test_batch_generate_chat_response(chat_lm: LanguageModel) -> None:
+    responses = chat_lm.batch_generate_chat_response([[{"role": "user", "content": "こんにちは。"}]], max_length=40)
```

Comment:
Tests failed because running `batch_generate_chat_response` with the model without a chat template.